src/model/request/CeloMintErc721.ts

```diff
@@ -9,4 +11,10 @@ export class CeloMintErc721 extends MintErc721 {
   @IsNotEmpty()
   @IsIn([Currency.CELO, Currency.CUSD])
   public feeCurrency: Currency;
+  @IsOptional()
+  @ValidateIf(o => (o.authorAddresses && o.cashbackValues) || !o.cashbackValues)
+  public authorAddresses?: string[];
+  @IsOptional()
```

Comment:
should be isNotEmpty

Reply:
move this logic to base class MintErc721. It should be present in every child in 1 place.